setup.py

```diff
@@ -7,8 +7,8 @@
 PACKAGE_VERSION = '0.1'
 
 deps = [
-    'manifestparser',
-    'marionette-client',
+    'manifestparser == 0.9',
+    'marionette-client == 0.8.6',
```

Comment:
I was told to add those dependencies in case your own package makes use of it. Given that this is not the case so far, we can indeed remove this dependency.